public/src/actions/account_info.jsx

```diff
@@ -10,11 +10,12 @@ export function setAccountInfo(user) {
 
 export function fetchUser(id) {
   return dispatch => axios.get(`/users/${id}`, {
-    headers: { 'Authorization': ` JWT ${localStorage.getItem('auth_token')}` }
+    headers: { Authorization: ` JWT ${localStorage.getItem('auth_token')}` }
   })
   .then((res) => {
     const userData = res.data;
-    return dispatch(setAccountInfo(userData));
+    dispatch(setAccountInfo(userData));
+    return userData;
```

Comment:
This was intended. Returning dispatch was ultimately returning undefined. The dispatched action still runs, but I figured might as well return the user Data for testing purposes... and for good measure. Did not have an observable affect on functionality. Open to changing this though.